meson.build

```diff
@@ -0,0 +1,30 @@
+project('sensors', 'cpp',
+    default_options: [
+        'warning_level=3',
+        'werror=true',
+        'cpp_std=c++17',
+    ],
+    license: 'Apache-2.0',
+    version: '2.0',
+)
+
+conf = configuration_data()
+
+conf.set_quoted('MAPPER_BUS', get_option('mapper-bus'))
+conf.set_quoted('MAPPER_PATH', get_option('mapper-path'))
+conf.set_quoted('MAPPER_IFACE', get_option('mapper-iface'))
+conf.set_quoted('SYSTEMD_PROPERTIES', get_option('systemd-properties'))
+conf.set_quoted('SENSORS_PATH', get_option('sensors-path'))
+conf.set_quoted('SENSOR_VALUE_IFACE', get_option('sensor-value-iface'))
+
+conf.set('WITH_REMOTE_HOST', get_option('remote-host-support'))
+
+configure_file(output: 'config.h', configuration: conf)
+
+executable('sensors',
```

Comment:
Lets rename it to lssensors.
I will add the command `sensors` to CLI as alias for `sudo lssensors`.

Reply:
done